strapi/strapi
# feat: add content from relations to history versions endpoint

### What does it do?

- Saves only the ID in the data key when history versions are related _(this is the in progress part, the current approach doesn't really work)_
- On retrieval, for each relation, populate an object with results and missingCount
- This PR starts messing around with the frontend too (relation input in history), but it's mostly pseudocode to make sure the approach works. The actual implementation is a separate ticket.

### Why is it needed?

We need to display relations in history

### How to test it?

🚨 Because of the data structure change, your app may crash with pre-existing history versions 🚨

Create history versions with some relations. View history, you should see the data. Go delete the related content, you should see that too in the history frontend.
